pgjdbc/src/test/java/org/postgresql/test/jdbc2/ParameterStatusTest.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2019, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.test.jdbc2;
+
+import org.postgresql.PGConnection;
+import org.postgresql.core.ServerVersion;
+import org.postgresql.test.TestUtil;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import java.util.Map;
+import java.util.TimeZone;
+import java.util.logging.Logger;
+
+//
+// This test covers client interface for server ParameterStatus messages
```

Comment:
Javadoc should be `/**... *... */` style comments

Reply:
Can do. Lots of projects don't bother with JavaDoc in tests and I wasn't so bothered with the format there.